packages/terra-button/src/_mixins.scss

```diff
@@ -1,6 +1,6 @@
 //  Map Button Variant's Color Schemes
 // ==========================
-
+/* stylelint-disable scss/no-global-function-names */
 @mixin terra-button-variant-color($color-map) {
   background-color: map-get($color-map, background-color);
   border-color: map-get($color-map, border-color);
```

Comment:
what happens when we use 
`@use "sass:map";`
And replace map-get with `map.get()` ?

Reply:
This map-get code has been added long back around 6years for which getting lint error now ,So have just disabled this lint error .